app/main.py

```
@@ -1,6 +1,40 @@
+import os
+
+import requests
+
+API_KEY = os.getenv("API_KEY")
+URL = "http://api.weatherapi.com/v1/current.json"
+CITY = "Paris"
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    if not API_KEY:
+        raise ValueError("No API key provided for Weather API")
```

probably it's better to raise some custom error `MissingAPIKeyError` or smth like this